src/main/java/com/orange/lo/sdk/rest/model/CommandAddRequest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Orange. All Rights Reserved.
+ * <p>
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.orange.lo.sdk.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CommandAddRequest {
```

Comment:
why not using Lombok and `@Data` annotation for example ?
(lombok also build equals and toString for you)